app/services/campaign_creator.rb

```diff
@@ -8,11 +8,14 @@ def self.run(params)
 
   def initialize(params)
     @params = params.slice(:name)
+    @campaign_id = params[:campaign_id] || raise(ArgumentError)
   end
 
   def run
     client.create_multilingual_campaign(@params).tap do |response|
-      if response.code != 201
+      if response.code == 201
```

Comment:
The check against the response code predates this PR, but since you're here, what about changing this to `if response.success?`